enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -1547,7 +1565,7 @@ func (c *FirecrackerContainer) Create(ctx context.Context, actionWorkingDir stri
 	if err != nil {
 		return err
 	}
-	if err := os.MkdirAll(c.getChroot(), 0755); err != nil {
+	if err = os.MkdirAll(c.getChroot(), 0755); err != nil {
```

Comment:
(nit: revert this diff)

```suggestion
	if err := os.MkdirAll(c.getChroot(), 0755); err != nil {
```